### PR TITLE
#1067-part2-Deserialize json string when suboption '-s genJson' is specified

### PR DESCRIPTION
**Pull request by Wxin-Duan**
When user specify '-s genJson', umple should not only produce toJson() to serialize Json string, but also fromJson() to deserialize Json string and instantiate validate objects.

### CODE REVIEW

File: UmpleToJava/UmpleTLTemplates/toJson_declare.ump [hunk 1]
```diff
@@ -180,17 +190,7 @@ class UmpleToJava {
...
-  * Generate Json for this object and connected objects visited objects to enable avoidance of infinite loops
-  *
-  * @param toJsonOutput  Output is aded to this as the network of objects is traversed
-  * @param visitedList  Every concrete object visited is added so we don't re-outpu
-  * @param nestLevel    As we output deeper objects, indent them more
-  * @param atConcreteClass false when we are recursing to a superclass
-  *     so we get the superclass data
-  * @return whether or not anything was output (so we can tell whether we need to output a comma)
-  */  
+  toJson_declare <<!<</*toJson_declare*/>>
```
Comment (TimLethbridge) [inline UmpleToJava/UmpleTLTemplates/toJson_declare.ump:209]: Why did you remove this block of comments for the toJson?

Comment (Wxin-Duan) [reply UmpleToJava/UmpleTLTemplates/toJson_declare.ump:209]: I moved this comment to the very top of the class, because the @return statement explains what toJsonHelper returns.

Comment (TimLethbridge) [reply UmpleToJava/UmpleTLTemplates/toJson_declare.ump:209]: Ah I see. It was in the wrong place! ... but there should be a comment generated before toJson (although it would not have any param descriuptions)

Comment (Wxin-Duan) [reply UmpleToJava/UmpleTLTemplates/toJson_declare.ump:209]: For sure, I just added a comment before toJson describe it would return a string in Json format
